__tests__/Json_encode_test.ml

```diff
@@ -2,6 +2,8 @@ open Jest
 open Expect
 open! Json.Encode
 
+external undef : Js.Json.t = "undefined" [@@bs.val]
```

Comment:
Might be better to just use `Obj.magic Js.undefined` instead of redefining it here.